### DISCUSSION TITLE
Discord Ban

### DISCUSSION THREAD

**Discussion by theyavuzarslan** [availproject/avail#316] [Q&A] [answered]
Hey everyone, 
I am waiting for participating to Clash of Nodes but I am banned from Discord - I dont have any idea why, can you help me fix this? 

All the best,
Thanks

**Comment by Grraarrgghh** [comment]
What's your Discord ID/name?

**Reply by theyavuzarslan** [reply]
Swiss Army Knife#1438

**Comment by Grraarrgghh** [comment, marked as answer]
You were banned for excessive spam; 20+ messages in a 5 minute interval, including deleting another 5-10 messages to try and scrub the spam which were logged.

**Reply by theyavuzarslan** [reply]
I was chatting with people and probably deleted messages are from bot in faucet channel.

**Reply by theyavuzarslan** [reply]
Is it possible to add me again?
Server says link is invalid or expired.

**Reply by theyavuzarslan** [reply]
@Grraarrgghh any update?

**Comment by BullForAReason** [comment]
I'm not able to join discord can someone let me know what's wrong ?
